features/old/signup/spam_protection.feature

```diff
@@ -24,6 +24,15 @@ Feature: Spam protection for buyer signup
      And I fill in the signup fields as "hugo"
      And I should not see the captcha
 
+  Scenario: Captcha is set to auto
```

Comment:
We should not change `features/old` anymore. Instead, create a new `.feature` file ideally matching the route of the view being tested.

In this file, please try using a more declarative style as described at https://cucumber.io/docs/bdd/better-gherkin/. For instance something like:
```cucumber
Given a provider with spam protection set for suspicious only
When a buyer wants to sign up
Then the buyer doesn't need to pass the captcha if signup form is filled wrong
But the buyer will need to pass the captcha if signup form is filled in too quickly
```

Reply:
I tried to remain close to your proposal, although I see some problems. First, IMO what is considered "good" or "bad" cucumbers is subjective and ambiguous, I'm not sure to be able to do it right next time. Also, the last two steps are so specific that they can't be reused. If we always write the tests this way, we'll en up having a few huge `*_steps.rb` files containing tens of steps that are only being called from one place.